### PR TITLE
clarify project philosophy and guidelines

### PR DESCRIPTION
**Pull request by rtribotte**
This pr add documentation to clarify project philosophy and guidelines.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,6 +55,36 @@ You have to deploy CRDs manually with the following command:
...
 ```
 
+### Exposing the Traefik dashboard
+
+This HelmChart do not expose the Traefik dashboard by default, for security concerns.
+Thus, there are multiple ways to expose the dashboard. 
+For instance, the dashboard access could be achieved through a port-forward :
+
+```
+kubectl port-forward pod/my-traefik-pod 9000:9000
```
Comment (rtribotte) [inline README.md:65]: @mmatur Is this command better ?
```
kubectl port-forward $(kubectl get pods --selector "app.kubernetes.io/name=traefik" --output=name) 9000:9000
```

File: traefik/README.md [hunk 2]
```diff
@@ -43,6 +55,36 @@ You have to deploy CRDs manually with the following command:
 kubectl apply -f traefik/crds
 ```
 
+### Exposing the Traefik dashboard
+
+This HelmChart do not expose the Traefik dashboard by default, for security concerns.
```
Comment (SantoDE) [inline traefik/README.md:60]: do => does?

File: README.md [hunk 3]
```diff
@@ -8,6 +8,18 @@ microservices with ease.
 This chart bootstraps Traefik version 2 as a Kubernetes ingress controller,
 using Custom Resources `IngressRoute`: <https://docs.traefik.io/providers/kubernetes-crd/>.
 
+### Philosophy
+
+The Traefik HelmChart is focused on Traefik deployment configuration.
+
+To keep this HelmChart as generic as possible we tend 
+to avoid integrating any third party nor any specific use cases.
```
Comment (SantoDE) [inline README.md:16]: third party solutions (?)

File: README.md [hunk 4]
```diff
@@ -43,6 +55,36 @@ You have to deploy CRDs manually with the following command:
 kubectl apply -f traefik/crds
 ```
 
+### Exposing the Traefik dashboard
+
+This HelmChart do not expose the Traefik dashboard by default, for security concerns.
```
Comment (SantoDE) [inline README.md:60]: does not expose ?

File: README.md [hunk 5]
```diff
@@ -43,6 +55,36 @@ You have to deploy CRDs manually with the following command:
...
+Thus, there are multiple ways to expose the dashboard. 
+For instance, the dashboard access could be achieved through a port-forward :
+
+```
+kubectl port-forward pod/my-traefik-pod 9000:9000
+```
+
+Another way would be to apply your own configuration, for instance,
+by defining and applying an ingressRoute CRD (`kubectl apply -f dashboard.yaml`):
```
Comment (SantoDE) [inline README.md:69]: IngressRoute ?

File: traefik/README.md (hunk 2 above)
Comment (rtribotte) [reply traefik/README.md]: @SantoDE applied !

**Pull request merged**